src/tool_storage.rs

```diff
@@ -344,11 +364,10 @@ impl ToolStorage {
                     .interact_opt()?;
 
                 if let Some(false) | None = proceed {
-                    eprintln!(
-                        "Skipping installation of {} and exiting with an error.",
-                        name
+                    bail!(
+                        "Tool {name} is not trusted. \
+                         Run `aftman trust {name}` in your terminal to trust it.",
```

Comment:
Note that we no longer exit the process here because that would make "greedy" installation of as many trusted tools as possible, impossible. We still bail and that will prevent any callers of `trust_check` from using it incorrectly. I assume this was done for safety reasons but since there are lints for unhandled `Result` types it should be hard to miss if anything needs to use `trust_check` in the future.